### PR TITLE
Added some more setup instructions to the README

### PR DESCRIPTION
**Pull request by nhajratw**
These were steps I needed to do to get my publishing to work.

### CODE REVIEW

**Comment by derklaro** [comment]
That signing plugin is documented like shit...

**Review by marcphilipp — APPROVED** [review]
Thanks, LGTM!

@szpak WDYT?

File: README.md [hunk 1]
```diff
@@ -66,6 +66,56 @@ nexusPublishing {
 }
 ```
 
+#### Configure Signing ####
```
Comment (marcphilipp) [inline README.md:69]: Maybe add a link to https://docs.gradle.org/current/userguide/signing_plugin.html#sec:signing_publications?

**Comment by DenuxPlays** [comment]
This needs to be merged!
It helped me so much...

File: README.md (hunk 1 above)
Comment (szpak) [reply README.md:69]: Added manually.

**Pull request merged**